src/collections/_documentation/workflow/releases/health.md

```diff
@@ -0,0 +1,137 @@
+
+### iOS
+To benefit from the health data provided in Releases v2, you must use at least version 5.0.0 Beta 2 of the Cocoa SDK and enable the collection of the release health metrics by adding `“enableAutoSessionTracking": true` into options during the initialization of the SDK.
+
+For more details, see the [full documentation on using Release Health with iOS]({%- link _documentation/platforms/cocoa/index.md -%}#release-health).
+
+## Data Filtering
+
+Sentry has inbound data filters. These filters are exclusively applied at ingest time and not later in processing. This, for instance, lets one discard an error by error message when the error is ingested through the JSON API. On the other hand, this filter doesn't apply to ingested minidumps.
+Data filters are only partially supported for session events, and they might be working slightly different:
+
+### Equivalent Filters
+
+**release filter**
+This filter discards events of a particular release. If sessions are started or updated and would match this filter, sessions are silently discarded entirely. This is consistent with error events.
+
+**web crawler filter**
+When session events send their user agent, they are discarded the same way as error events are. There might be a slight difference in the fallback case where the user agent of the sending SDK is used (browser JavaScript).
+
+**legacy browsers**
+This works exactly like the web crawler filter. No sessions are created or updated for legacy browsers when enabled.
+
+**ip filter**
+Throws away sessions of a specific IP.
+
+#### Unsupported Filters
+
+**error message filter**
+This filter does not apply for sessions. Even if a user is discarding such errors, they will count against their stability score. An alternative measure *could be* to discard such sessions entirely if an error of this type is filtered out, but we believe this would make the system harder to understand.
+
+**browser extensions**
+Since the logic for detecting such errors requires all the error information, we cannot support this filter at the session ingest time. If we want to improve this case, we could retroactively discard such sessions entirely once we identify such an error case.
+
+## Release Health Glossary
+
+#### Session
+
+The time when the user is interacting with the application. (The application is in foreground)
+
+- Session begins with the start of the application. Or, it begins with bringing the already started application back from background to the foreground.
+
+- Session ends with the closing of the application or with the application being sent to the background.
+
+- If the application is in the background for less than 30 seconds, we do not need to start the session again.
+
+- Applications that are active even on the background (for example, music player) should track the sessions separately for the background process.
```

Comment:
I'd not say separately, but manually.
it's important to note that using session health automatically and manually together won't work.
its either automatic or manual.